src/reportPerf.ts

```diff
@@ -31,5 +31,13 @@ export const reportPerf = (
       rating,
       navigationType,
     });
-  });
+  };
+
+  // Send CLS and INP metrics immediately,
+  // because this metrics are reported when page is hidden or closed
+  if (['CLS', 'INP'].includes(measureName)) {
```

Comment:
@SuperOleg39 would you mind add tests for this fix please.

Reply:
Done!